shopfloor_delivery_shipment/services/delivery_shipment.py

```diff
@@ -746,8 +769,15 @@ def _find_move_lines_from_product(
         """
         domain = self._find_move_lines_domain(shipment_advice)
         domain.extend(
-            [("product_id", "=", product.id), ("picking_id", "=", picking.id)]
+            [
+                ("product_id", "=", product.id),
+                ("picking_id", "=", picking.id),
```

Comment:
if picking

Reply:
The scenario does not allow to search lines for a product if a transfer has not been scanned before. So not needed.